src/components/EventForm/EventForm.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Paper from '@material-ui/core/Paper';
+import Button from '@material-ui/core/Button';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+import FormBody from './FormBody';
```

Comment:
I'm confused, which file do you want to use? You imported Form and FormBody

I suggest that you delete one of them completely